src/libraries/Common/src/System/Diagnostics/NetFrameworkUtils.cs

```diff
@@ -63,7 +63,7 @@ private static bool SafeWaitForMutex(Mutex mutexIn, ref Mutex mutexOut)
         private static bool SafeWaitForMutexOnce(Mutex mutexIn, ref Mutex mutexOut)
         {
             bool ret;
-            RuntimeHelpers.PrepareConstrainedRegions();
+
```

Comment:
This .cs file is consumed by _System.Diagnostics.PerformanceCounter.csproj_ and _System.Diagnostics.EventLog.csproj_, both of which target netstandard + netframework. The call to `PrepareConstrainedRegions` should remain for those targets. Might be easiest just to pragma suppress this instance.

Reply:
These projects consume these files under `$(TargetFramework.StartsWith('$(NetCoreAppCurrent)')) or $(TargetFramework.StartsWith('netcoreapp2.0'))` condition, so it should be fine to remove it.